docs/tasks.md

```diff
@@ -684,3 +685,46 @@ Except as otherwise noted, the content of this page is licensed under the
 [Creative Commons Attribution 4.0 License](https://creativecommons.org/licenses/by/4.0/),
 and code samples are licensed under the
 [Apache 2.0 License](https://www.apache.org/licenses/LICENSE-2.0).
+
+## Debugging a Task
```

Comment:
can you put a link to this in the TOC somewhere at the top of the file? might even be good to link from teh main README